### PR TITLE
fix (example) : fix menu and size div for globe example

### PR DESCRIPTION
**Pull request by gchoqueux**

### CODE REVIEW

File: examples/globe.html [hunk 1]
```diff
@@ -1,31 +1,18 @@
...
-                height: 100%;
-            }
-
-            body {
-                margin: 0;
-                overflow:hidden;
-                height:100%;
-            }
-
+            html {height: 100%}
```
Comment (peppsac) [inline examples/globe.html]: Why did you change the above css rules?

File: examples/globe.html [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
+            body { margin: 0; overflow:hidden; height:100%}
             #viewerDiv {
                 margin : auto auto;
-                width: 800px;
-                height: 600px;
+                width: 100%;
+                height: 100%;
```
Comment (peppsac) [inline examples/globe.html:19]: This is the only css change that is needed.

File: examples/globe.js [hunk 3]
```diff
@@ -8,19 +8,27 @@ const positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25
 const viewerDiv = document.getElementById('viewerDiv');
 
 // Instanciate iTowns GlobeView*
-const globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer });
+const globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+const menuGlobe = new GuiTools('menuDiv');
+menuGlobe.view = globeView;
```
Comment (peppsac) [inline examples/globe.js]: This change breaks the test. `{ renderer }` is mandatory here.

**Review by peppsac — DISMISSED** [review]

File: examples/globe.js (hunk 3 above)
Comment (autra) [reply examples/globe.js]: And nits: `new GuiTools('menuDiv', globeView);` is enough :-)

**Comment by gchoqueux** [comment]
@peppsac updated

File: examples/globe.html [hunk 4]
```diff
@@ -34,19 +34,13 @@
...
     <body>
         <div id="viewerDiv"></div>
-
         <script src="GUI/GuiTools.js"></script>
         <script src="../dist/itowns.js"></script>
         <script src="../dist/debug.js"></script>
         <script type="text/javascript">
             let renderer;
-            let exports = {};
+            let exports = {};
```
Comment (autra) [inline examples/globe.html]: nits, blank at the end of the line

File: examples/globe.js [hunk 5]
```diff
@@ -9,18 +9,26 @@ const viewerDiv = document.getElementById('viewerDiv');
 
 // Instanciate iTowns GlobeView*
 const globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer });
+const menuGlobe = new GuiTools('menuDiv');
```
Comment (autra) [inline examples/globe.js]: and I think `new GuiTools('menuDiv', globeView);` will work

Comment (gchoqueux) [reply examples/globe.js]: not yet

Comment (autra) [reply examples/globe.js]: oh ok, yeah, maybe a `this.view = view` should be added. Not worth the trouble here, we'll do it later.

**Review by autra — DISMISSED** [review]
LGTM, I'm waiting for travis :-)

File: examples/globe.html [hunk 6]
```diff
@@ -44,9 +43,18 @@
         </script>
         <script src="globe.js"></script>
         <script type="text/javascript">
+            /* global itowns, document, GuiTools, globeView, promises */
             const menuGlobe = new GuiTools('menuDiv');
-            new debug.Debug(globeView, viewerDiv);
```
Comment (autra) [inline examples/globe.html:48]: Is the suppression of debug voluntary?

Comment (autra) [reply examples/globe.html:48]: Actually yes, I'm not sure they will work on gh-pages because dist is not compiled with debug flag. Good to me.

**Review by autra — APPROVED** [review]
